README.md

```diff
@@ -320,6 +320,33 @@ class MyLeapGame extends LeapGame {
 }
 ```
 
+## Debugging
+
+### Slow motion
+
+`LeapWorld` includes
+[`HasTimeScale`](https://pub.dev/documentation/flame/latest/components/HasTimeScale-mixin.html),
+so you can set `world.timeScale = 0.5` to slow your whole game down to 50% speed to make it easier
+to play test nuanced bugs. (You can use this as slow motion for your game too.)
+
+### Render hitbox
+
+`PhysicalEntity` includes a `debugHitbox` property you can override which will automatically
+draw a box indicating the exact hitbox the collision detection system is using for your
+entity.
+
+```dart
+class MyPlayer extends PhysicalEntity {
+
+  @override
+  void onUpdate(double dt) {
```

Comment:
I believe this should be `update` right? I don't remember Leap naming the update method differently than flame haha